doc/source/parameters/cfd/laser_heat_source.rst

```diff
@@ -30,44 +31,74 @@ If a laser heat source is present in a simulation, it can be added in this secti
   end
 
 
-* The ``enable`` parameter is set to true if the problem has a laser heat source term and enables its calculation.
+* The ``enable`` parameter is set to ``true`` if the problem has a laser heat source term and enables its calculation.
 
-* Laser ``concentration factor`` parameter indicates the definition of the beam radius. In almost all the articles, it is assumed equal to 2.0.
+* The ``type`` parameter is set to ``exponential decay`` (default) if the media hit by the incident laser ray has a non-flat topography (e.g. powder bed). This enables the inclusion of the ``penetration depth`` parameter in the heat flux calculation. If the surface is flat, the ``type`` could be set at ``material interface vof`` and used in conjunction with the :doc:`VOF auxiliary physic <./volume_of_fluid>`. The different models are detailed :ref:`below <LaserTypes>`.
```

Comment:
I am not sure that's an adequate way of formualting it. Because if the surface is flat y ou could also use the surface model. The main thing here is how we want to model it, a volume contribution or surface contribution. I would not comment on the influence of the surface topology here :)

Reply:
That was my understanding of why one would use a _volumetric flux_ instead of the _surface flux_, but it's true that both could be used. I'll change the formulation avoiding this explanation and simply explaining the difference as one being a volumetric source and the other a surface one!